modin/pandas/test/test_io.py

```diff
@@ -143,7 +152,11 @@ def eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
     unique_filename_pandas = get_unique_filename(extension=extension)
 
     try:
-        getattr(modin_obj, fn)(unique_filename_modin, **fn_kwargs)
+        try:
+            getattr(modin_obj, fn)(unique_filename_modin, **fn_kwargs)
+        except EXCEPTIONS:
+            getattr(modin_obj, fn)(unique_filename_modin, **fn_kwargs)
```

Comment:
Pls add a comment on why is this needed.
Also I suggest turning this in a `for _ in range(3)` (or some other number) loop.